qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -302,11 +358,31 @@ def init(args):
 
     spell.init()
 
-    _init_profiles()
+    app = QApplication.instance()
+
+    log.init.debug("Initializing qute://* handler...")
+    global _qute_scheme_handler
+    _qute_scheme_handler = webenginequtescheme.QuteSchemeHandler(parent=app)
+
+    log.init.debug("Initializing request interceptor...")
+    global _req_interceptor
+    host_blocker = objreg.get('host-blocker')
+    args = objreg.get('args')
+    _req_interceptor = interceptor.RequestInterceptor(host_blocker, args=args,
+                                                      parent=app)
+
+    log.init.debug("Initializing QtWebEngine downloads...")
+    global _download_manager
+    _download_manager = webenginedownloads.DownloadManager(parent=app)
+    objreg.register('webengine-download-manager', _download_manager)
+
+    global default_profile, private_profile
+    default_profile = create_profile()
+    private_profile = create_profile(private=True)
     config.instance.changed.connect(_update_settings)
 
     global global_settings
-    global_settings = WebEngineSettings(_SettingsWrapper())
+    global_settings = WebEngineSettings(_SettingsWrapper)
```

Comment:
with the `@staticmethod` changes above, this would be an instance rather than a class again.